benchmark/feldera-sql/benchmarks/nexmark/queries/q21.rs

```diff
@@ -0,0 +1,26 @@
+use dashmap::DashMap;
+use once_cell::sync::Lazy;
```

Comment:
https://doc.rust-lang.org/std/cell/struct.OnceCell.html this doesn't work yet?

Reply:
Thanks, I changed it to use [`LazyLock`](https://doc.rust-lang.org/std/sync/struct.LazyLock.html)